projects/hitl/tests/conftest.py

```diff
@@ -1,7 +1,18 @@
 import pytest
```

Comment:
Remove this folder (`projects/hitl/tests`) for now? I thought we decided to put tests in a different folder?